### PR TITLE
[ingress-nginx][docs] Improve inlet docs

### PR DESCRIPTION
**Pull request by unreturned**
## Description
Added info that field is immutable
Added specific info about switching from this inlet (temporary solution until implementation https://github.com/deckhouse/deckhouse/issues/7552)

## Why do we need it, and what problem does it solve?
Improve documentation

## Why do we need it in the patch release (if we do)?
There's no need.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ingress-nginx
type: chore
summary: Improve inlet documentation.
impact_level: low
```

### CODE REVIEW

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ spec:
                       **Внимание!** На одном хосте может быть только один контроллер с данным типом inlet'а.
 
                       **Внимание!** Необходимо, чтобы на узле были свободны следующие порты: 80, 81, 443, 444, 10354, 10355.
+
+                      **Внимание!** При переходе с данного inlet на какой-либо другой необходимо произвести зачистку правил iptables с последующим перезапуском подов kube-proxy, либо вовсе перезагрузить ноды, на которых размещались ingress-контроллеры.
```
Comment (elenashliaga) [inline modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:31]:
```suggestion
                      **Внимание!**  Чтобы поменять инлет, удалите правила iptables и перезапустите поды kube-proxy или перезагрузите узлы, на которых размещались Ingress-контроллеры.
```

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec:
                     **Важно!** Если указать значение "nginx", дополнительно будут обрабатываться Ingress-ресурсы без аннотации `kubernetes.io/ingress.class` или поля `spec.ingressClassName`.
                 inlet:
                   description: |
-                    Способ поступления трафика из внешнего мира:
+                    Способ поступления трафика из внешнего мира. Неизменяемый в течение жизни ресурса параметр.
```
Comment (elenashliaga) [inline modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:18]:
```suggestion
                    Способ приема трафика из внешней сети. Параметр в течение жизни ресурса изменить не получится.
```

File: modules/402-ingress-nginx/crds/ingress-nginx.yaml [hunk 3]
```diff
@@ -40,7 +40,7 @@ spec:
                 inlet:
                   type: string
                   description: |
-                    The way traffic goes to cluster from the outer network.
+                    The way traffic goes to cluster from the outer network. This field is immutable after creation.
```
Comment (elenashliaga) [inline modules/402-ingress-nginx/crds/ingress-nginx.yaml:43]:
```suggestion
                    The way traffic from the external network is routed to the cluster. Once you have set the method, you cannot change it later.
```

File: modules/402-ingress-nginx/crds/ingress-nginx.yaml [hunk 4]
```diff
@@ -52,6 +52,8 @@ spec:
                       **Caution!** There can be only one controller with this inlet type on a host.
 
                       **Caution!** The following ports must be available on the node: 80, 81, 443, 444, 10354, 10355.
+
+                      **Caution!** When switching from this inlet to any other, it is necessary to clear the iptables rules and then restart the kube-proxy pods, or completely reboot the nodes on which the ingress-controllers were located.
```
Comment (elenashliaga) [inline modules/402-ingress-nginx/crds/ingress-nginx.yaml:56]:
```suggestion
                      **Caution!** To change inlet, remove the iptables rules and restart the kube proxy pods or reboot the nodes hosting ingress controllers.
```

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml (hunk 2 above)
Comment (z9r5) [reply modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:18]: Выбрав способ?

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml (hunk 1 above)
Comment (z9r5) [reply modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:31]: @elenashliaga А тут не выделяем kube-proxy?

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml (hunk 2 above)
Comment (MariaMakeeva) [reply modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:18]: А что, если написать просто: **Способ приема трафика из внешней сети. Изменение выбранного способа невозможно.**

**Review by z9r5 — APPROVED** [review]

**Pull request merged**